lib/membrane/rtcp/receiver_reporter.ex

```diff
@@ -0,0 +1,154 @@
+defmodule Membrane.RTCP.ReceiverReporter do
+  @moduledoc """
+  Periodically generates RTCP receive reports basing on jitter buffer stats and RTCP sender reports
+  and sends them via output.
+  """
+  use Membrane.Source
+  require Logger
+  alias Membrane.{Buffer, RTCP, RTP, Time}
+
+  def_output_pad :output, mode: :push, caps: :any
+
+  def_options interval: [
+                type: :time,
+                default: 5 |> Time.seconds(),
+                description: """
+                Time interval between requesting stats for reports.
+                Reports are sent when stats are collected.
+                """
+              ],
+              sender_report_timeout: [
+                type: :time,
+                default: 60 |> Time.seconds(),
+                description: """
+                Minimal time between receiving RTCP sender report
+                and assuming it won't be needed anymore.
+                """
+              ]
```

Comment:
This seems like a dirty fix rather than proper solution. The parent should inform about ssrc pad removal.

In general, I have the feeling that half of this element should not be an element but rather a module used by receiver bin. The responsibility of this element could just push the RTCP reports and Receiver Bin, instead of forwarding numerous messages to the Reporter would simply call functions.

Imagine that we wanted a proper report scheduling based on bandwidth - that would be yet another set of messages (some kind of bandwidth reports) to forward here in order to calculate when the report should be sent